### PR TITLE
Fix Navbar issue, change the positioning, and add several new buttons

### LINKED ISSUE #443: State Table Generator Not Accessible in Mobile
## Describe the bug
The State Table Generator in flipflop2 page needs to be changed as it is available currently on desktop view. If the User wants to access it through mobile then it asks for pressing F and there is no keyboard that pops up on page load. 

## To Reproduce

1. Go to 'https://learn.circuitverse.org/docs/flipflop2.html'
2. Use Mobile View
3. See error

## Expected behavior
We need to change the press F functionality to a button, so that the user can start the table generation or the keyboard should pop up on page load in mobile view.

## Screenshots
Desktop View : 
image

Mobile View :
image

## Desktop (please complete the following information):

* OS: [Windows 10]
* Browser [Chrome]

## Smartphone (please complete the following information):

* OS: [Android]
* Browser [Chrome]

## Additional context

The Navbar needs to fixed in the Mobile View as it is a bit shifted from its original positionas in the other pages. 

I am not working on the Issue. Please comment below to work on it.

### PR DESCRIPTION
**Pull request by b30wulffz**
Fix Navbar issue, change the positioning, and add several new buttons

Fixes #443  

#### Changes done:
- [x] Fixed Navbar issue
- [x] Added buttons to restart the generator, and to go back to about section
- [x] Fixed positioning of elements
- [x] Changed UI
- [x] Updated JS
- [x] Reformatted the code using Prettier

#### Screenshots

mobile

pc

#### Preview Link(s): 
[https://deploy-preview-444--tender-ardinghelli-ebb79f.netlify.app/](https://deploy-preview-444--tender-ardinghelli-ebb79f.netlify.app/)

#### ✅️ By submitting this PR, I have verified the following

- [x] Checked to see if a similar PR has already been opened 🤔️
- [x] Reviewed the contributing guidelines 🔍️
- [x] Sample preview link added (add the link(s) for all the pages changed/updated from the checks tab after checks complete)
- [ ] Tried Squashing the commits into one

### CODE REVIEW

**Comment by b30wulffz** [comment]
@Shivansh2407 @YashKumarVerma 
Please Review

File: _includes/flipflop2.html [hunk 1]
```diff
@@ -1,359 +1,501 @@
...
+          output = generateStateTable(
+            flipflopCount,
+            inputCount,
+            flipflops,
+            input["flipflop"]
+          );
+
+          var table = "<p><table class='table'><thead>";
+
+          // //table += "<th colspan=" + flipflopCount + ">Present State</th><th colspan=" + inputCount + ">Input</th><th colspan=" + flipflopCount + ">Next State</th><th colspan=" + flipflops.length + ">FlipFlop Inputs</th>";
```
Comment (YashKumarVerma) [inline _includes/flipflop2.html:424]: consider removing it ?

Comment (b30wulffz) [reply _includes/flipflop2.html:424]: Was commented by someone else. Hence didn't remove. Should I?

Comment (abhinavsri360) [reply _includes/flipflop2.html:424]: @b30wulffz I think you should

**Comment by b30wulffz** [comment]
@YashKumarVerma 
Made changes, will squash after receiving confirmation.

**Comment by b30wulffz** [comment]
@Shivansh2407 Any updates regarding this PR?

**Comment by YashKumarVerma** [comment]
Thanks a lot for your contributions.

**Comment by Shivansh2407** [comment]
Nice Work @b30wulffz 👍 
We Also need to fix the table generated after filling the drop-downs, as sometimes we get undefined. An alert should be displayed instead of showing undefined in the table. We need to do that in a different PR.

**Pull request merged**